Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -1819,7 +1829,7 @@ namespace Remotion.BocAutoCompleteReferenceValue
         const availableWidth = position.left + referenceWidth - marginLeft;
         // js rounding errors sometimes create linebreaks
         // therefore we add a single pixel for the field to always be wide enough:
-        const requiredWidth = contentWidth + 1;
+        const requiredWidth = contentWidth + 1 + (requiredHeight >= maxHeightSafe ? scrollbarWidth : 0);
```

Comment:
maxHeightSafe: did you verify that it really is `>= maxHeightSafe` and not `> maxHeightSafe`?

Reply:
I tried it, and it only works with >=, when looking at the code afterwards it seems that most cases occur where required height is minimized between `maxHeightSafe` and some other value. Therefore `==` would actually make most sense semantically, but as that does not make much sense to understanding whats happening, `>=` seems like the better option.

Reply:
okay, so basically it's the same weirdness we have with the `+1` for the width? Don't you just love fractal (okay, fractional. or is it fractured?) layout engines?